
No_Login_rest_access_groupdel.py::test_start[0] 
-------------------------------- live log call ---------------------------------
INFO     root:No_Login_rest_access_groupdel.py:16 Trying -> /rest/access/groupdel
INFO     root:waiter.py:26 Waiting rest response to be changed...
INFO     root:waiter.py:26 Waiting rest response to be changed...
INFO     root:waiter.py:26 Waiting rest response to be changed...
FAILED

=================================== FAILURES ===================================
________________________________ test_start[0] _________________________________

debugLevel = '0'

    @allure.title("No login _rest_access_groupdel method check")
    def test_start(debugLevel):
        debugEnable(int(debugLevel))
        link = "/rest/access/groupdel"
        host = getFromConfig("host")
        logging.info("Trying -> " + link)
        request0 = waitGetResponseToBeChanged("http://" + host + link, "data")
        try:
>           request = waitGetResponseToBeChanged("http://" + host + link, "data").split('"message":"')[1].split('"')[0]
E           IndexError: list index out of range

No_Login_rest_access_groupdel.py:19: IndexError

During handling of the above exception, another exception occurred:

debugLevel = '0'

    @allure.title("No login _rest_access_groupdel method check")
    def test_start(debugLevel):
        debugEnable(int(debugLevel))
        link = "/rest/access/groupdel"
        host = getFromConfig("host")
        logging.info("Trying -> " + link)
        request0 = waitGetResponseToBeChanged("http://" + host + link, "data")
        try:
            request = waitGetResponseToBeChanged("http://" + host + link, "data").split('"message":"')[1].split('"')[0]
        except:
>           request = waitGetResponseToBeChanged("http://" + host + link, "data").split('"message": "')[1].split('"')[0]
E           IndexError: list index out of range

No_Login_rest_access_groupdel.py:21: IndexError
------------------------------ Captured log call -------------------------------
INFO     root:No_Login_rest_access_groupdel.py:16 Trying -> /rest/access/groupdel
INFO     root:waiter.py:26 Waiting rest response to be changed...
INFO     root:waiter.py:26 Waiting rest response to be changed...
INFO     root:waiter.py:26 Waiting rest response to be changed...
=========================== short test summary info ============================
FAILED No_Login_rest_access_groupdel.py::test_start[0] - IndexError: list ind...
========================= 1 failed in 70.44s (0:01:10) =========================
